web/src/components/KebabMenu.tsx

```diff
@@ -114,7 +115,14 @@ const KebabMenu = ({ items, show, setShow, Footer }: KebabMenuProps): ReactEleme
               </IconButton>
             </ActionBar>
           </Heading>
-          <Content>{items}</Content>
+          <Content>
+            {items.map((item, index) => (
+              <React.Fragment key={`menu-item-${index + 1}`}>
+                {item}
+                {index < items.length - 1 && <Divider />}
+              </React.Fragment>
+            ))}
+          </Content>
```

Comment:
💭 Same here, maybe we can use a Mui list here?

Also, we do previously have a bottom border on the last element as well.

Reply:
check production... there is no divider on the last element.
<img width="312" height="336" alt="image" src="https://github.com/user-attachments/assets/89660b92-47cb-42e7-9c55-a31b6a00dd27" />